### PR TITLE
System.Resources cleanup

### PR DESCRIPTION
**Pull request by JeremyKuhne**
Some non-functional cleanup of the System.Resources code. The first commit just breaks the classes out. The second changes (most of) the field names to be per standard. The third just does some simple spacing fixes and catches on classes fields I missed in the second.

cc: @KlausLoeffelmann 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/3434)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Resources/AssemblyNamesTypeResolutionService.cs [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        {
+            _names = names;
+        }
+
+        public Assembly GetAssembly(AssemblyName name)
+        {
+            return GetAssembly(name, true);
+        }
+
+        //
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Resources/AssemblyNamesTypeResolutionService.cs:35]: Your regexp missed this comment :)

File: src/System.Windows.Forms/src/System/Resources/ResxFileRef.Converter.cs [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    {
+        public class Converter : TypeConverter
+        {
+            public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
+            {
+                if (sourceType == typeof(string))
+                {
+                    return true;
+                }
+                return false;
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Resources/ResxFileRef.Converter.cs:27]: FYI: Not sure if your intention was to insert blank lines before the return statements, but I see it was done in another file, but not in this one.

File: src/System.Windows.Forms/src/System/Resources/AssemblyNamesTypeResolutionService.cs (hunk 1 above)
Comment (JeremyKuhne) [reply src/System.Windows.Forms/src/System/Resources/AssemblyNamesTypeResolutionService.cs:35]: Whoops, thanks!

File: src/System.Windows.Forms/src/System/Resources/ResxFileRef.Converter.cs (hunk 2 above)
Comment (JeremyKuhne) [reply src/System.Windows.Forms/src/System/Resources/ResxFileRef.Converter.cs:27]: There is supposed to be a blank line after every closing bracket. I'll scour it again.

**Comment by JeremyKuhne** [comment]
Note that I have a bigger change pending- wanted to get a good chunk of the simpler bits out to make it easier to review/iterate on the next PRs.

**Comment by JeremyKuhne** [comment]
@Tanya-Solyanik I updated most of the missed ones.

**Comment by Tanya-Solyanik** [comment]
>@Tanya-Solyanik I updated most of the missed ones.

Thank you!

**Pull request merged**